include/nccl_ofi_tuner.h

```diff
@@ -21,16 +21,16 @@ OFI_NCCL_PARAM_INT(tuner_num_channels, "TUNER_NUM_CHANNELS", 8);
  * combine the parameters. This parameter is meant for internal testing only and
  * is not meant to be documented for users.
  */
-OFI_NCCL_PARAM_INT(tuner_net_latency, "TUNER_NET_LATENCY", 20);
+OFI_NCCL_PARAM_INT(tuner_net_latency, "TUNER_NET_LATENCY", 75);
```

Comment:
this doesn't make sense to me.

Reply:
I agree that there's more work to be done here in the model itself, but given the current model this provides a better fit than what we have today.